doc/v2/_static/petab_schema_v2.yaml

```diff
@@ -22,96 +22,67 @@ properties:
       File name (absolute or relative) or URL to PEtab parameter table
       containing parameters of all models listed in `problems`. A single
       table may be split into multiple files and described as an array here.
-  problems:
-    type: array
-    description: |
-      One or multiple PEtab problems (sets of model, condition, observable
-      and measurement files). If different model and data files are
-      independent, they can be specified as separate PEtab problems, which
-      may allow more efficient handling. Files in one problem cannot refer
-      to models entities or data specified inside another problem.
-    items:
 
-      type: object
-      description: |
-        A set of PEtab model, observable and measurement
-        files and optional condition, experiment, and visualization files.
-
-      properties:
-
-        model_files:
-          type: object
-          description: One or multiple models
-
-          # the model ID
-          patternProperties:
-            "^[a-zA-Z_]\\w*$":
-              type: object
-              properties:
-                location:
-                  type: string
-                  description: Model file name or URL
-                language:
-                  type: string
-                  description: |
-                    Model language, e.g., 'sbml', 'cellml', 'bngl', 'pysb'
-              required:
-                - location
-                - language
-          additionalProperties: false
-
-        measurement_files:
-          type: array
-          description: List of PEtab measurement files.
-
-          items:
-            type: string
-            description: PEtab measurement file name or URL.
-
-        condition_files:
-          type: array
-          description: List of PEtab condition files.
+  model_files:
+    type: object
```

Comment:
why is this `object` but `array` for the other files? Should this not be `array`

Reply:
So far it's:
```yaml
model_files:
  some_id:
    location: ...
    language: ...
```
This could be changed to 

```yaml
model_files:
  - id: some_id
    location: ...
    language: ...
```

I am fine with either, but it's out of scope of this PR.